misc/docker/README.md

```diff
@@ -0,0 +1,203 @@
+# ObsPy Docker Utilities
+
+This directory contains a collections of scripts and Dockerfiles enabling
+developers to..
+
+ * test ObsPy on various Unix based operating systems, and
+ * build ObsPy deb packages on Debian/Ubuntu distributions
+
+It requires a working installation of [Docker](https://www.docker.com/). It has
+been designed to work with a remote Docker installation so it works fine on OSX
+and I guess Windows as well.
+
+
+## 1. ObsPy Test Runner on Docker Images
+
+The `run_obspy_tests.sh` script will test either
+
+ * the **current state of the repository** on the specified Docker image, or..
+ * test a clean github clone at a certain commit from either the main
+   repository or a fork.
+
+Test reports will be sent automatically to
+[http://tests.obspy.org](http://tests.obspy.org).
+
+If the respective image is not yet available it will be created automatically.
+The `base_images` directory contains all available images receipts.
+
+```bash
+$ ls base_images
+centos_7                debian_8_jessie   opensuse_13_2          ubuntu_14_04_trusty
+debian_7_wheezy         fedora_22         opensuse_leap          ubuntu_15_10_wily
+debian_7_wheezy_32bit   fedora_23         ubuntu_12_04_precise
```

Comment:
Minor nitpick: This is waaay out of date by now especially with your additions in this PR ;-)